### DISCUSSION TITLE
Remind users that they aren't allowed to name this extension on Scratch

### DISCUSSION THREAD

**Discussion by Dhuls-Scratch** [ScratchAddons/ScratchAddons#3201] [Ideas]
If you search up Scratch Addons on Scratch (for convenience go [here](https://scratch.mit.edu/search/projects?q=Scratch+Addons)) you will see a lot of projects that violate [this policy](https://scratch.mit.edu/discuss/topic/284272/?page=1) by advertising this extension (except for one, which was made before this extension). In addition, there is always the occasional topic in the forums complaining why the filter bot censors it, and a lot of people try to bypass the filter bot. Maybe a warning upon installing (preferably linking to the announcement) would help lower the amount of users breaking Scratch's rules, and getting alerts/bans.

**Comment by apple502j** [comment]
We currently do that on comments, BaG and Japanese forums. Maybe we can do the same for every forums, with options to hide it.

**Reply by WorldLanguages** [reply]
> there is always the occasional topic in the forums complaining why the filter bot censors it

@Dhuls-Scratch Are you sure that is still the case? We already do what you're suggesting in comments in order to provide more context to our users on why the filter bot would mute them if they really want to post the comment.

**Reply by CST1229** [reply]
They could use a different spelling that doesn't trip our filter at all
